wavelength-plugins/dice.js

```diff
@@ -115,7 +116,7 @@ class Dice {
 	}
 
 	end(user) {
-		if (user) this.room.add('|uhtmlchange|' + this.room.diceCount + '|<div class = "infobox">(This game of dice has been forcibly ended by ' + Chat.escapeHTML(user.name) + '.)</div>').update();
+		if (user) this.room.add('|uhtmlchange|' + this.room.diceCount + '|<div class = "infobox">(This game of dice has been forcibly ended by ' + WL.nameColor(user.name) + '.)</div>').update();
```

Comment:
Id rather not use name color for something like this, the box isnt supposed to draw attention since its ended.

Reply:
Noted